src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -22,6 +22,7 @@ import { UserList } from "components/molecules/UserList";
 import { RoomModalOngoingEvent, ScheduleItem } from "..";
 
 import "./RoomModal.scss";
+import { logEventGA } from "utils/ga";
```

Comment:
Reorder import.

Reply:
fixed in bdf4d19497739d4a2ed30c0616c136ef109529f9